stages/org.osbuild.anaconda

```diff
@@ -48,6 +72,25 @@ def main(tree, options):
         for m in modules:
             f.write(f"    {m}\n")
 
+        # The other module lists are optionally supplied
+        if "activatable-modules" in options:
+            f.write("activatable_modules =\n")
+
+            for m in options["activatable-modules"]:
+                f.write(f"    {m}\n")
+
+        if "forbidden-modules" in options:
+            f.write("forbidden_modules =\n")
+
+            for m in options["forbidden-modules"]:
+                f.write(f"    {m}\n")
+
+        if "optional-modules" in options:
+            f.write("optional_modules =\n")
+
+            for m in options["optional-modules"]:
+                f.write(f"    {m}\n")
+
```

Comment:
If the configuration strings contain newlines, this will all be screwed. But we have that all over the place already. I know.

Reply:
Good point though, we could (should?) add patterns to the respective strings.